tests/snuba/api/endpoints/test_organization_sessions.py

```diff
@@ -1428,3 +1428,470 @@ def test_release_is_empty(self):
                     "totals": {"sum(session)": 1},
                 }
             ]
+
+
+@patch("sentry.api.endpoints.organization_sessions.release_health", MetricsReleaseHealthBackend())
```

Comment:
I realize the tests are lengthy and could probably be refactored but honestly it will take away from the readability of the tests, and I am a fan of writing verbose tests